props/saber_fett263_buttons.h

```diff
@@ -4653,7 +4723,11 @@ SaberFett263Buttons() : PropBase() {}
 	  break;
 	}
         if (!SaberBase::Lockup()) {
-          SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+          if (fusor.angle1() < - M_PI / 4) {
+            SaberBase::SetLockup(SaberBase::LOCKUP_DRAG);
```

Comment:
This doesn't seem to have anything to do with choreography.

Reply:
Will submit separately